common/views/components/IIIFViewer/MainViewer.tsx

```diff
@@ -305,6 +305,35 @@ const ItemRenderer = memo(({ style, index, data }: ItemRendererProps) => {
 
 ItemRenderer.displayName = 'ItemRenderer';
 
+export function scrollViewer(
```

Comment:
Why do we export this into `GridViewer` rather than just having it there?

Reply:
we use it [here](https://github.com/wellcomecollection/wellcomecollection.org/pull/6854/files#diff-28343f384fd03b2a55160366889a77a28eaa4c1b8f93edf8a716cba16b15f5c3R386) too